Formula/z.lua.rb

```diff
@@ -0,0 +1,30 @@
+class ZLua < Formula
+  desc "New cd command that helps you navigate faster by learning your habits"
+  homepage "https://github.com/skywind3000/z.lua"
+  url "https://github.com/skywind3000/z.lua/archive/1.8.4.tar.gz"
+  sha256 "ae26580b5e0a3b3d82a932804e0d2cd9ead44cbb49edb3360d3c00d4ae12f06b"
+  head "https://github.com/skywind3000/z.lua.git"
+
+  bottle :unneeded
+
+  depends_on "lua"
+
+  def install
+    pkgshare.install "z.lua", "z.lua.plugin.zsh", "init.fish"
+    libexec.install "test_path.lua"
+    doc.install "README.md", "LICENSE"
+  end
+
+  def caveats
```

Comment:
Most of this should be in the upstream documentation. You could reference the install location since that's brew specific (and therefore an acceptable caveat)

Reply:
Please check if this new caveat simple enough? Thanks.

Reply:
It shouldn't exist at all, users are expected to read the upstream documentation without the formula reminding them. Caveats should only exist to tell people about things that are non-standard due to the install with homebrew.

Reply:
Caveats have been deleted.